EXPLAINER.md

```diff
@@ -53,6 +53,10 @@ post-it notes, anyone can read them, and unless read-only, anyone can write
 them. Web NFC does not handle automatic encryption or signing of NFC content
 (with NDEF signatures): handling PKI infrastructure is left to applications.
 
+The API does not support controlling who can change the content on an NDEF
+tag. However, the API supports making an NDEF tag permanently read-only, see the
+[examples](https://w3c.github.io/web-nfc/#make-an-nfc-tag-permanently-read-only).
+
 ## API shape and examples
```

Comment:
Shall we add a simple JS example in this section?

```js
const ndef = new NDEFReader();
try {
  await ndef.write("Hello world");
  console.log("Message written.");
  await ndef.makeReadOnly();
  console.log("NFC tag has been made permanently read-only after writing to it.");
} catch (error) {
  console.log(`Operation failed: ${error}`);
}
```

Reply:
I thought that referring to the examples in the spec was fine, but we can have it inline, too, indeed.

Reply:
Actually in other sections we also only link the examples.
Inline examples are there only for the integrations.
I think it makes sense this way (linking examples).

Reply:
I think you should be able to read an explainer without having to open or link into other documents

Reply:
Should we inline the other examples as well, then?

Reply:
I would say yes